test-galasactl-local.sh

```diff
@@ -0,0 +1,259 @@
+                                          ;;
+        galasactl-linux-s390x )           binary="galasactl-linux-s390x"
+                                          ;;
+        galasactl-windows-amd64.exe )     binary="galasactl-windows-amd64.exe"
+                                          ;;
+        -h | --help )                     usage
+                                          exit
+                                          ;;
+        * )                               error "Unexpected argument $1"
+                                          usage
+                                          exit 1
+    esac
+    shift
+done
+
+if [[ "${binary}" == "" ]]; then
+    error "Need to specify which binary of galasactl to use."
+    usage
+    exit 1  
+fi
+
+#--------------------------------------------------------------------------
+# Initialise Galasa home
+function galasa_home_init {
+    h2 "Initialising galasa home directory"
+
+    mkdir -p ${BASEDIR}/temp
+    cd ${BASEDIR}/temp
+
+    cmd="${BASEDIR}/bin/${binary} local init \
+    --log -"
+
+    info "Command is: $cmd"
+
+    $cmd
+    rc=$?
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to initialise galasa home"
+        exit 1
+    fi
+    success "Galasa home initialised"
+}
+
+#--------------------------------------------------------------------------
+# Invoke the galasactl command to create a project.
+function generate_sample_code {
+    h2 "Invoke the tool to create a sample project."
+
+    cd $BASEDIR/temp
+
+    export PACKAGE_NAME="dev.galasa.example.banking"
+    ${BASEDIR}/bin/galasactl-darwin-amd64 project create --package ${PACKAGE_NAME} --features payee,account --obr --maven --gradle --force
+    rc=$?
+    if [[ "${rc}" != "0" ]]; then
+        error " Failed to create the galasa test project using galasactl command. rc=${rc}"
+        exit 1
+    fi
+    success "OK"
+}
+
+#--------------------------------------------------------------------------
+# Now build the source it created using maven
+function build_generated_source_maven {
+    h2 "Building the sample project we just generated."
+    cd ${BASEDIR}/temp/${PACKAGE_NAME}
+    mvn clean test install 
+    rc=$?
+    if [[ "${rc}" != "0" ]]; then
+        error " Failed to build the generated source code which galasactl created."
+        exit 1
+    fi
+    success "OK"
+}
+
+#--------------------------------------------------------------------------
+# Now build the source it created using gradle
+function build_generated_source_gradle {
+    h2 "Building the sample project we just generated."
+    cd ${BASEDIR}/temp/${PACKAGE_NAME}
+    gradle build publishToMavenLocal
+    rc=$?
+    if [[ "${rc}" != "0" ]]; then
+        error " Failed to build the generated source code which galasactl created."
+        exit 1
+    fi
+    success "OK"
+}
+
+#--------------------------------------------------------------------------
+# Run test using the galasactl locally in a JVM
+function submit_local_test {
```

Comment:
I wonder if, now we've got the galasactl method of calling a test, whether we need to test this java -jar method ?